andela/workdey
# #123003821 Ft skillset search for taskees

# [#123003821] Skillset search for taskees
#### What does this PR do?

This PR implements the functionality to search for taskees given a required skillset
#### Description of Task to be completed?

In this task, anytime a tasker wants to know the number of taskees that has a particular skillset, all he needs to do is to enter the skillset on a search bar on his profile page
#### How should this be manually tested?

Visit the site and login as a tasker. Enter the a skillset in the search bar that can be found on the page header. You should be directed to a page that lists all the taskees that has that particular skillset.
#### What are the relevant pivotal tracker stories?
#123003821
